packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -1,6 +1,5 @@
-import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import capitalize from 'lodash-es/capitalize';
-import orderBy from 'lodash-es/orderBy';
+import React, { type ReactElement, type ReactNode, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { capitalize, lowerCase } from 'lodash-es';
 import { useTranslation } from 'react-i18next';
 import { useReactToPrint } from 'react-to-print';
 import {
```

Comment:
Looks like we can remove the `Tag` component import as we're no longer using it.